airflow_example/dags/tableau_datasource_update/tableau_datasource_update.py

```diff
@@ -147,12 +159,12 @@ def __set_connection_attributes(self):
 
         return {
             'conn_type': 'snowflake',
-            'conn_db': snowflake_conn.extra_dejson['database'],
-            'conn_schema': snowflake_conn.extra_dejson['schema'],
-            'conn_host': f'{snowflake_conn.host}.snowflakecomputing.com',
-            'conn_role': snowflake_conn.extra_dejson['role'],
-            'conn_user': snowflake_conn.login,
-            'conn_warehouse': snowflake_conn.extra_dejson['warehouse']
+            'conn_db': SNOWFLAKE_DATABASE,
+            'conn_schema': SNOWFLAKE_SCHEMA,
+            'conn_host': SNOWFLAKE_URL,
+            'conn_role': SNOWFLAKE_ROLE,
+            'conn_user': SNOWFLAKE_USER,
+            'conn_warehouse': SNOWFLAKE_WAREHOUSE
```

Comment:
Can we revert this?

Reply:
You should be able to pull these attributes from the snowflake conenction using the `SnowflakeHook`.
`from airflow.providers.snowflake.hooks.snowflake import SnowflakeHook`
i.e.
```python
{
    'conn_type': 'snowflake',
    'conn_db': snowflake_hook.database,
    'conn_schema': snowflake_hook.schema,
    'conn_host': f'{snowflake_hook.account}.snowflakecomputing.com',
    'conn_role': snowflake_hook.role,
    'conn_user': snowflake_hook.user,
    'conn_warehouse': snowflake_hook.warehouse
}
```